MMA.md

```diff
@@ -0,0 +1,365 @@
+  "client": "ios",
+  "token": "yFmAZ4G4TReVcJY8rjhtVudo98xHyEELsIlbSM2UcaY",
+  "event": "E_Cleared_Private_Data",
+  "userId": "A3AF645D-129F-40DB-AF49-FBCA556B1541",
+  "value": "0.000000",
+  "action": "track",
+  "devMode": true,
+  "deviceId": "A3AF645D-129F-40DB-AF49-FBCA556B1541",
+  "sdkVersion": "2.0.0-SNAPSHOT"
+}
+~~~~
+
+Download videos or any other media
+~~~~
+{
+  "time": "1500062287.263575",
+  "client": "ios",
+  "token": "yFmAZ4G4TReVcJY8rjhtVudo98xHyEELsIlbSM2UcaY",
+  "event": "E_Download_Media_Saved_Image",
+  "userId": "A3AF645D-129F-40DB-AF49-FBCA556B1541",
+  "value": "0.000000",
+  "action": "track",
+  "devMode": true,
+  "deviceId": "A3AF645D-129F-40DB-AF49-FBCA556B1541",
+  "sdkVersion": "2.0.0-SNAPSHOT"
+}
+~~~~
+
+Save password and login from door hanger
+~~~~
+{
+  "time": "1500062287.263575",
+  "client": "ios",
+  "token": "yFmAZ4G4TReVcJY8rjhtVudo98xHyEELsIlbSM2UcaY",
+  "event": "E_Saved_Login_And_Password",
+  "userId": "A3AF645D-129F-40DB-AF49-FBCA556B1541",
+  "value": "0.000000",
+  "action": "track",
+  "devMode": true,
+  "deviceId": "A3AF645D-129F-40DB-AF49-FBCA556B1541",
+  "sdkVersion": "2.0.0-SNAPSHOT"
+}
+~~~~
+
+Deep Links:
+Deep links are actions that can point Firefox to open certain pages or load features such as `show bookmark list` or
+`open Firefox Account Settings`. When users receives a Message, they can click the button(s) on it. These buttons can
+trigger the following deep links:
+* Link to sync signup/sign in (firefox://deep-link?url=settings/fxa)
+* Link to default search engine settings (firefox://deep-link?url=settings/search)
+* Link to bookmark list (firefox://deep-link?url=homepanel/bookmarks)
+* Link to history list (firefox://deep-link?url=/history)
+* Link to general preferences (firefox://deep-link?url=settings)
+* Link to the mailto preferences (firefox://deep-link?url=settings/mailto)
+* Link to open a new tab (firefox://deep-link?url=settings/newtab)
+* Link to open the user's homepage (firefox://deep-link?url=settings/homepage)
+* Link to open the settings page so the user can clear their private data (firefox://deep-link?url=settings/clear-private-data)
+* Link to open a new private tab (firefox://deep-link?url=homepanel/new-private-tab)
+
+
+Note these deep links can only be processed from within Firefox (i.e., if they are clicked from outside of Firefox, Firefox will open but nothing further will happen).
+
+Messages :
+Messages are in-app prompts to the user from Leanplum. The user interaction of that prompt will be sent to the Leanplum backend (such as "Accept" or "Show") to track overall engagement with the Message. The Message is downloaded from Leanplum when the Leanplum SDK is initialized at App start, assuming the fulfillment criteria for the Message is met. As mentioned before, the fulfillment criteria is a set of required Events and User Attributes. The fulfillment criteria are set in the Leanplum backend.
+
+The List of current messages are:
```

Comment:
Looking at the campaigns listed here, I realize that they're the Android campaigns - there are fewer for iOS.

However, since we have the documentation by Jean and Nevin on the wiki already, let's just remove this whole section and link to that wiki page instead, saying something like "The current messages are documented on our Leanplum Contextual Hints wiki page" linking to https://wiki.mozilla.org/Leanplum_Contextual_Hints . That way, this part won't get out of date because there will be a single place to track it.